web/src/sagas/index.js

```diff
@@ -25,9 +27,13 @@ function* fetchClientVersion() {
 }
 
 function* fetchZoneHistory(action) {
-  const { zoneId } = action.payload;
+  const { zoneId, timescale } = action.payload;
   try {
-    const payload = yield call(protectedJsonRequest, `/v3/history?countryCode=${zoneId}`);
+    let path = `/v3/history?countryCode=${zoneId}`;
+    if (timescale !== TIMESCALE.LIVE) {
+      path += `&timescale=${timescale}`;
```

Comment:
I see that the `TIMESCALE.MONTHLY` constant gets translated to `MONTHLY` instead of a duration format when making the API call: `/history?countryCode=IE&timescale=MONTHLY`

Wouldn't it be nicer if the backend could accept any integer duration in seconds (or at least duration string formats like `5y`, `10d`, `24h`) to make it work with more general durations?

If that seems over-engineered, I think something like `timescale=5y&step=1mo` / `timescale=24h&step=15m` would still be a nicer API (even if those were the only two supported options) :slightly_smiling_face: